src/components/RangeSeries.tsx

```diff
@@ -26,27 +27,47 @@ export function RangeSeries<T extends RangeSeriesPoint>(
 ) {
   const id = useId(props.id, 'series');
   const [, legendDispatch] = useLegend();
-  const { lineStyle = {}, hidden, xAxis, yAxis, data, label } = props;
-
+  const {
+    lineStyle = {},
+    hidden,
+    xAxis = 'x',
+    yAxis = 'y',
+    data,
+    label,
+    xShift: oldXShift = '0',
+    yShift: oldYShift = '0',
+  } = props;
+
+  const { xShift, yShift } = useShift({
+    horizontalAxisId: xAxis,
+    verticalAxisId: yAxis,
+    xShift: oldXShift,
+    yShift: oldYShift,
+  });
   // Update plot context with data description
   const dispatch = usePlotDispatchContext();
+  const { xShift: xShiftInverted, yShift: yShiftInverted } = useInvertShift({
+    xShift,
+    yShift,
+  });
   useEffect(() => {
     const [xMin, xMax] = extent(data, (d) => d.x);
 
     const [y1Min, y1Max] = extent(data, (d) => d.y1);
     const [y2Min, y2Max] = extent(data, (d) => d.y2);
 
-    const x = { min: xMin, max: xMax, axisId: xAxis };
+    const x = { min: xMin, max: xMax, shift: xShiftInverted, axisId: xAxis };
     const y = {
       min: Math.min(y1Min, y2Min),
       max: Math.max(y1Max, y2Max),
+      shift: -yShiftInverted,
```

Comment:
Because in the plot we start calculating px from the top left 
So 
Scale.invert(value)-Scale.invert(0)
Will work in Xshift but not in yshift

But i can put the - in the hook (i think it will be better)